lib/create-testing-library-rule.ts

```diff
@@ -0,0 +1,37 @@
+import { ESLintUtils, TSESLint } from '@typescript-eslint/experimental-utils';
+import { getDocsUrl } from './utils';
+import {
+  detectTestingLibraryUtils,
+  DetectionHelpers,
+} from './detect-testing-library-utils';
+
+type CreateRuleMetaDocs = Omit<TSESLint.RuleMetaDataDocs, 'url'>;
+type CreateRuleMeta<TMessageIds extends string> = {
+  docs: CreateRuleMetaDocs;
+} & Omit<TSESLint.RuleMetaData<TMessageIds>, 'docs'>;
```

Comment:
This is copied from `@typescript-eslint/experimental-utils`, I don't know if I could inferred that somehow.

Reply:
I think this is fine, we should perhaps add the same comment in the code (that it's copied from there).
It might be useful in the future 😅

Reply:
Good idea, I'm gonna add it now.